autogen/docs/behaviorspace.md.mustache

```diff
@@ -337,6 +337,9 @@ will be always be taken at the end of each model run, even if this text box is
   would record every tenth tick, as well as the last tick. Multiple reporters
    can be combined using *and* and *or*.
 
+**Pre experiment commands:** These commands will be run once, before the
+experiment begins.
```

Comment:
I'm not 100% sure how to phrase it, but I think we need to point out that these pre/post experiment commands cannot affect the state of any of the runs (well, they can, but only the "primary" one).  To try a bit:

> These commands execute outside of the actual runs, so they cannot affect the global variables or agents in the model (use the **Setup commands** and **Post run commands** for that).  If you use primitives like `file-open` or the CSV extension in the pre-experiment commands, their internal state will not carry over into the runs and errors will likely occur if used there.